core/src/main/scala/com/nvidia/spark/rapids/tool/Platform.scala

```diff
@@ -191,7 +283,12 @@ abstract class Platform(var gpuDevice: Option[GpuDevice],
     "3.4.2" -> "342",
     "3.4.3" -> "343",
     "3.5.0" -> "350",
-    "3.5.1" -> "351"
+    "3.5.1" -> "351",
```

Comment:
Updated list of supported shuffle managers ([link](https://docs.nvidia.com/spark-rapids/user-guide/latest/additional-functionality/rapids-shuffle.html#supported-spark-versions))

Reply:
Created issue - https://github.com/NVIDIA/spark-rapids-tools/issues/1676 for automating generation of supported shims